android/src/main/java/com/iterable/reactnative/Serialization.java

```diff
@@ -198,6 +199,20 @@ static IterableConfig.Builder getConfigFromReadableMap(ReadableMap iterableConte
                 configBuilder.setLogLevel(logLevel);
             }
 
+            if(iterableContextJSON.has("dataRegion")) {
+                int dataRegion = iterableContextJSON.getInt("dataRegion");
+                switch (dataRegion) {
+                    case 1:
+                        dataRegion = IterableDataRegion.EU;
```

Comment:
0 is US and 1 is EU in native. Will be good to have an explicit case check for 0. Default can still stay.

Reply:
Makes sense